src/main/resources/errors.yaml

```diff
@@ -152,7 +163,80 @@ server-errors:
       
       ${SNIPPET.RETRY}
   - scope: DATABASE
+    code: DRIVER_TIMEOUT
+    http-status-override: 504
+    title: Database connection was timed out while processing the request
+    body: |-
+      The Data API connection to the database was timed out by the database while processing the request.
+
+      Writing to the ${schemaType} ${keyspace}.${table} failed to complete successfully. If this request modified data the changes may have been written to by some replicas, but not all. Future read requests may return eventually consistent results.
+
+      The detailed response from the database was: ${errorMessage}    
+
+      ${SNIPPET.RETRY}
+  - scope: DATABASE
+    code: NO_NODE_AVAILABLE
+    http-status-override: 502
+    title: Database connection was timed out while processing the request
+    body: |-
+      No node was available to execute the query.
+
+      Writing to the ${schemaType} ${keyspace}.${table} failed to complete successfully. 
+
+      The detailed response from the database was: ${errorMessage}    
+
+      ${SNIPPET.RETRY}
+  - scope: DATABASE
+    code: READ_FAILURE
+    http-status-override: 502
+    title: Database read failed
+    body: |-
+      The Data API failed to read from the database.
+
+      Reading from the ${schemaType} ${keyspace}.${table} failed to complete successfully. 
+
+      The request was was waiting for ${blockFor} replicas to acknowledge the read, but only ${received} replicas responded, ${numFailures} failed.
+
+      ${SNIPPET.RETRY}
+  - scope: DATABASE
+    code: READ_TIMEOUT
+    http-status-override: 504
+    title: Database read timed out
+    body: |-
+      The Data API timed out while reading from the database.
+
+      Reading from the ${schemaType} ${keyspace}.${table} failed to complete successfully. 
+      #TODO(Hazel): ReadTimeoutException has multiple ways to format the error message. What message should we use?
```

Comment:
Probably don't want to have comment within text block? (it will be included as text, not comment)

Reply:
Yes, will remove it later - just want to leave the comments at the nearest place